dali/test/python/test_operator_gaussian_blur.py

```diff
@@ -223,6 +228,11 @@ def test_generic_gaussian_blur():
         for (t_in, t_out) in [(np.uint8, types.NO_TYPE), (np.float32, types.FLOAT), (np.uint8, types.FLOAT)]:
             yield from generate_generic_cases(dev, t_in, t_out)
 
+def test_one_sized_extent():
+    for dev in ["cpu", "gpu"]:
+        for shape, layout in [((10, 1, 3), "HWC"), ((1, 10, 3), "HWC"), ((1, 10), "HW"), ((10, 1), "HW")]:
+            yield check_generic_gaussian_blur, 10, 2.0, 5, shape, layout, 2, dev, np.float32, types.FLOAT, False
```

Comment:
Add a 3D test without channels, with the outermost extent of 1.

Reply:
Should be equivalent to the 2D case with non-innermost extent equal one, but I can add one just in case.